python/ray/data/tests/test_logging_dataset.py

```diff
@@ -54,16 +55,21 @@ def _test_harness(filename_01: str, filename_02: str):
         f"Dataset {dataset_02_id} execution finished"
         in log_01_contents + log_02_contents
     )
+    assert "Worker function is running" in log_01_contents + log_02_contents
 
 
 def test_dataset_logging_concurrent(ray_start_regular_shared, reset_logging):
     from concurrent.futures import ThreadPoolExecutor
 
     def _short(x):
+        logger = logging.getLogger("ray.data")
+        logger.info("Worker function is running")
         time.sleep(0.1)
         return x
 
     def _long(x):
+        logger = logging.getLogger("ray.data")
+        logger.info("Worker function is running")
```

Comment:
can we print different logs and check that they are actually printed in the right files?